src/UI/SignTransactionsModals/SignWithDeviceModal/index.tsx

```diff
@@ -32,14 +33,24 @@ const SignWithDeviceModal = ({
     container: 'card container',
     cardBody: 'card-body'
   });
+  const { show, hide } = useDappModal();
```

Comment:
maybe these should be named "showModal, hideModal", so it's easier to read in the code